### ISSUE TITLE
install error with lastest version (2.0.0.009-SNAPSHOT-DSM6)

### ISSUE THREAD

**Issue by eriu** [openhab/openhab-syno-spk#49]
Hi,
I downloaded the lastest release version (https://github.com/openhab/openhab-syno-spk/releases/tag/2.0.0.009-SNAPSHOT-DSM6) and tried to install it on my ds214. But there is a error at the download part.

The problem is, that the download link is brocken (404 error, Not Found)
> https://openhab.ci.cloudbees.com/job/openHAB-Distribution/lastSuccessfulBuild/artifact/distributions/openhab/target/openhab-2.0.0-SNAPSHOT.zip

this link works fine:
> https://openhab.ci.cloudbees.com/job/openHAB-Distribution/lastSuccessfulBuild/artifact/distributions/openhab/target/openhab-2.1.0-SNAPSHOT.zip

thanks

**Comment by cniweb** [comment]
Yes, thanks for your Issue. I change the download URL to the new release.

**Comment by Speedliner** [comment]
> The problem is, that the download link is brocken (404 error, Not Found)

Same problem here. I was able to manually download the file openhab-2.1.0-SNAPSHOT.zip, but couldn't use that one for the installation.

**Comment by sheindel** [comment]
I have also run into this issue, and since it just unzips and copies the file, I downloaded it, renamed it, and put it into my public folder, and it still failed the installation. So the "local file" install it not working on my Synology (DS213j with DSM 6.0) and the only thing I can figure is that it can't successfully run synoshare without being an elevated user. When I ran synoshare at the command line as an elevated user, I had no problems running the declarations and pre install portion of the script, but it still would not work from the installer.
